src/types/annotation.ts

```diff
@@ -0,0 +1,11 @@
+export interface PostAnnotationResponse {
+  stream: string
+  'start-time': string
+  'end-time': string
+  summary: string
+  stickers?: any
+}
```

Comment:
oh, one more thing I forgot to mention: since we're only implementing a single annotation and not ranges for the first part, `end_time` should always equal `start_time`

second: we camelCase our variables here, and we make sure external things conform to our standards. so we don't create variables called `start-time` because that's how they're passed into the API. We create a variable called `startTime` and then set `start-time` into `startTime` at the boundaries of the system when the value enters into our domain.

change `'start-time'` to `startTime`